pkg/hello/syncing.go

```diff
@@ -150,12 +158,13 @@ func NewSyncingClient(
 	transport SyncClientTransport,
 ) *SyncingClient {
 	return &SyncingClient{
-		myName:       myName,
-		nginxAdapter: nginxAdapter,
-		encoder:      encoder,
-		interval:     interval,
-		apps:         apps,
-		transport:    transport,
+		myName:               myName,
+		stateChangeGenerator: nginxAdapter,
+		encoder:              encoder,
+		interval:             interval,
+		apps:                 apps,
+		transport:            transport,
+		failureThreshold:     3,
```

Comment:
A candidate for const / configurable value through helm.

Reply:
Agreed, leaving for now